### DISCUSSION TITLE
Mia-Platform Console - Runtime section - Improve pod events list displayed age

### DISCUSSION THREAD

**Discussion by danibix95** [mia-platform/community#21] [Ideas]
Hi,

in Console Runtime section, once a pod is selected, it is possible to list all the K8s event related to it.   
I've been using such feature for a while and I often have a hard time to understand which is the events generation order, since multiple events are all notified within the same minute (as shown in the attached image).

Would it possible to show also the _number of seconds_ in the date displayed in `Age` column of the table?

![Screenshot from 2022-09-12 14-14-28](https://user-images.githubusercontent.com/3228581/189651329-cafea5ac-c372-4f77-bf5b-e609a16f5cac.png)

**Comment by giulioroggero** [comment]
Yes, for sure. We can it in the backlog of small enhancements. @davidebianchi

**Comment by ilteoood** [comment]
FYI, the events can be ordered by generation time clicking on the `age` column.

**Comment by akelity** [comment]
The seconds have been added to the Events tab in Runtime.
![image](https://github.com/user-attachments/assets/345f7aa9-e210-447f-b903-27b1b3972a50)
